src/dlgConnectionProfiles.cpp

```diff
@@ -119,7 +119,7 @@ dlgConnectionProfiles::dlgConnectionProfiles(QWidget * parent)
     } else {
         pWelcome_document->setHtml(QStringLiteral("<html><head/><body>%1</body></html>")
                                    .arg(tr("<p><center><big><b>Welcome to Mudlet!</b><bold></center></p>"
-                                           "<p><center><b>Click on one of the MUDs on the list to play.</b></center></p>"
+                                           "<p><center><b>Click on one of the games on the list to play.</b></center></p>"
```

Comment:
This PR is going to clash with #1952 ...! :stuck_out_tongue_winking_eye:

Reply:
That's OK, they can be merged.